### ISSUE TITLE
HTML button containing img with alt attribute should pass controlsWithoutLabel

### ISSUE THREAD

**Issue by RickSBrown** [GoogleChrome/accessibility-developer-tools#202]
I believe this should be fine from an a11y perspective:

`<button><img src="/foo.png" alt="Save"/></button>`

But it fails as the test below shows:

```
test('Button element with image with alt needs no label', function() {
    // Setup fixture
    var fixture = document.getElementById('qunit-fixture');

    var button = document.createElement('button');
    var img = button.appendChild(document.createElement('img'));
    img.setAttribute("alt", "Save");
    fixture.appendChild(button);

    var rule = axs.AuditRules.getRule('controlsWithoutLabel');
    equal(rule.run({ scope: fixture }).result,
          axs.constants.AuditResult.PASS);
});
```
